transcom/mymove
# B 20991 main - Add Counseling Office Column and Filtering to TIO Queue

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20991)

## Summary
When a customer selects an origin duty location assigned to government counseling, then they will also be asked to select a counseling office within the same GBLOC. This field will be called "Counseling Office".

How to test
1. Login as a customer and create a 2 both with different counseling offices.
2. Login as a Services Counselor and submit the new move.
3 Login as TOO, approve both moves.
4. Crete multiple payment requests for each move
5. Login as TIO and filter the Counseling Office Task Order Queue for a move to verify filtering works.
6. Login as TIO and sort the Counseling Office Task Order Queue for a move to verify sort works.

## Screenshots
[B-20991-508Screenshots.pdf](https://github.com/user-attachments/files/17740707/B-20991-508Screenshots.pdf)


## [Int PR](https://github.com/transcom/mymove/pull/14185)
